src/buttons/giveaway.ts

```diff
@@ -43,11 +51,19 @@ export default {
       await interaction.reply({ content: 'ERROR. Giveaway gibt es nicht', ephemeral: true })
       return
     }
+    const hash = dc_id + ga_id /* Sollte mal ein Hash sein, ist aber keiner (nicht wundern) */
+    await db.runAsync(`INSERT INTO giveaway_participants(dc_id, giveaway_message_id, hash) VALUES(?,?,?)`, [
+      dc_id,
+      ga_id,
+      hash
+    ])
+    await interaction.reply({ content: 'Du nimmst beim Giveaway teil', ephemeral: true })
```

Comment:
this is moved up from below, to make sure we always have the current participant count in the embed, without any calculations.